app/backend/src/couchers/models.py

```diff
@@ -2209,6 +2212,8 @@ def __str__(self):
     # any user creates any event, default to no notifications
     event__create_any = ("event:create_any", [], True, nd.EventCreate)
     event__update = ("event:update", dt_all, True, nd.EventUpdate)
+    event__cancel = ("event:cancel", dt_all, True, nd.EventCancel)
+    event__delete = ("event:delete", dt_all, True, nd.EventDelete)
```

Comment:
You will also need to add a code path in `notifications/render.py`.

Sorry the new notification setup isn't that well documented yet. I'm impressed you figured it out!

Reply:
Ah, render I missed - I was wondering where the Push notification is set :) Added it.